matminer/featurizers/tests/test_function.py

```diff
@@ -82,6 +87,21 @@ def test_helper_functions(self):
         self.assertTrue(parse_expr("x0 ** 2 / (x1 * x2)") in test_combo_3)
         self.assertEqual(len(test_combo_3), 8)
 
+        test_combo_4 = generate_expressions_combinations(["1 / x", "x ** 2", "exp(x)"],
+                                                         combo_function=np.sum)
+        self.assertEqual(len(test_combo_4), 9)
+        test_combo_4 = generate_expressions_combinations(["1 / x", "x ** 2", "exp(x)"],
+                                                         combo_function=lambda x: x[1]-x[0])
+        self.assertEqual(len(test_combo_4), 18)
+
+    def test_multi_featurizer(self):
+        ff1 = FunctionFeaturizer(expressions=["x ** 2"])
+        ff2 = FunctionFeaturizer(expressions=["exp(x)", "1 / x"])
+        mf = MultipleFeaturizer([ff1, ff2])
+        new_df = mf.featurize_dataframe(self.test_df, ['a', 'b', 'c'],
+                                        inplace=False)
+        self.assertEqual(len(new_df), 11)
+
 
```

Comment:
Can we add a test for this class behaving well in a Pipeline? That's where I want to use it (and figure that may be a common use case)

Reply:
I don't know what a Pipeline is.  Someone who will use it that way should probably add that test (and any of the necessary functionality).